### PR TITLE
Insights and Data API release notes for 2017-01-09

### PR DESCRIPTION
**Pull request by thallada**
We released a new version of Insights and the Analytics Data API today to prod and edge and would like to include the changes in the next release notes update.

### Date Needed (optional)

We released it today, so by the next release notes update would be best.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @dsjen 
- [x] Doc team review (sanity check and copy edit): @edx/doc
- [ ] Product review: @katymyw

FYI: Tag anyone else who might be interested in this PR here. @stroilova 

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2017/analytics/analytics_2017-01-09.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Update "student" to "learner" in the Insights guide text.
+
+Fixed a bug that caused Insights to throw a 500 error when a site-wide
```
Comment (dsjen) [inline en_us/release_notes/source/2017/analytics/analytics_2017-01-09.rst]: Nit: I think that this note might not be needed; it's sort of user-facing, but not totally relevant?

**Review by dsjen — APPROVED** [review]
Just one question about whether to include a point (no strong preferences).  LGTM!

File: en_us/release_notes/source/2017/analytics/analytics_2017-01-09.rst (hunk 1 above)
Comment (thallada) [reply en_us/release_notes/source/2017/analytics/analytics_2017-01-09.rst]: True, it's only relevant to Open Edx developers that may have encountered the bug. I'll omit this one.

**Comment by catong** [comment]
Doc 👍

**Comment by katymyw** [comment]
small question -- do we want to say "attempt to fix" in the release notes? (I know we're not certain). Probably fine to leave as is, but just wanted to ask. 

otherwise, 👍

**Comment by thallada** [comment]
@katymyw Yeah I wasn't totally sure what to put for that. We never confirmed that it actually fixed the bug the user was seeing, so it is an "attempt" still. Perhaps could find a better wording for that...

**Pull request merged**